antora.yml

```
@@ -16,6 +16,10 @@ asciidoc:
     webcomponent_url: https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@2/dist/tinymce-webcomponent.min.js
     jquery_url: https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@2/dist/tinymce-jquery.min.js
     default_meta_keywords: tinymce, documentation, docs, plugins, customizable skins, configuration, examples, html, php, java, javascript, image editor, inline editor, distraction-free editor, classic editor, wysiwyg
+    # product docker variables
+    dockerimageexporttopdf: registry.containers.tiny.cloud
```

@ShiridiGandham Looking at the other plugins, we might need to define the docker images such as `/pdf-converter`
```suggestion
    dockerimageexporttopdf: registry.containers.tiny.cloud/pdf-converter
```